packages/elements-core/src/utils/http-spec/IServer.ts

```diff
@@ -57,3 +61,28 @@ export const getServerUrlWithDefaultValues = (server: IServer): string | null =>
 
   return stringifiedUrl.endsWith('/') ? stringifiedUrl.slice(0, -1) : stringifiedUrl;
 };
+
+export const getServerUrlWithFilledVariables = (server: IServer, values?: { [key: string]: string }): string | null => {
```

Comment:
do we actually need that util at all? 
It does the same as `getServerUrlWithDefaultValues`, with the only difference being we optionally take values and apply them.
Feels like we could rename `getServerUrlWithDefaultValues` to `getServerUrlWithValues`, add `values`, read them, and that's it.